djangocms_alias/filters.py

```diff
@@ -82,6 +85,37 @@ def choices(self, changelist):
             }
 
 
+class CategoryFilter(admin.SimpleListFilter):
+    title = _("Category")
+    parameter_name = CATEGORY_FILTER_PARAM
+
+    def lookups(self, request, model_admin):
+        qs = model_admin.get_queryset(request)
+        cat_id = qs.values_list('alias__category', flat=True).distinct()
+        cat = Category.objects.filter(pk__in=cat_id).order_by('translations__name')
```

Comment:
```suggestion
        # Ensure the category is ordered by the name alphabetically by default
        cat = Category.objects.filter(pk__in=cat_id).order_by('translations__name')
```

Try to add comments to reinforce why something exists.